packages/one-app-bundler/webpack/loaders/common.js

```diff
@@ -25,6 +25,10 @@ const cssLoader = ({ name = '', importLoaders = 2 } = {}) => ({
     importLoaders,
     modules: {
       localIdentName: `${name && `${name}__`}[name]__[local]___[hash:base64:5]`,
+      // eslint-disable-next-line max-params, no-unused-vars
+      getLocalIdent: (loaderContext, localIdentName, localName, options) => (
```

Comment:
Im not sure what this is for as it seems super bundler specific. Can we get some code comments to describe the purpose of this function added?